static/scss/answers/theme.scss

```diff
@@ -180,4 +180,11 @@ $container-width-at-collapsed-filters-breakpoint: $breakpoint-collapsed-filters
   .yxt-SortOptions-optionLabel {
     font-size: var(--yxt-filters-and-sorts-font-size);
   }
+
+  .yxt-Card-child {
```

Comment:
would it be possible to scope this to just doc search?

Reply:
Inline snippets can appear on multiple different card types. Essentially, any card with a `details` section could be used to display such a highlighted snippet.